src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,84 @@
+package pl.javastart.task;
+
+public class Ticket {
+    private String event;
+    private Adress adress;
+    private double price;
+    private String type;
+    private static int ticketNo = 0;
+    private int countTicket;
+
+    public Ticket(String event, Adress adress, double price, String type) {
+        ticketNo++;
+        this.event = event;
+        this.adress = adress;
+        this.price = price;
+        this.type = type;
+        this.countTicket = ticketNo;
+
+    }
+
+    public Ticket() {
+        countTicket++;
+    }
```

Comment:
Ten konstruktor nie jest poprawny. Jeżeli z niego skorzystamy, to wszystkie bilety będą miały id=1. W drugim konstruktorze masz to rozwiązane poprawnie